components/style/properties/shorthand/position.mako.rs

```diff
@@ -374,7 +374,7 @@
             Either::First(ref areas) => {
                 let track_list = match *template_rows {
                     GenericGridTemplateComponent::TrackList(ref list) => list,
-                    _ => unreachable!(),        // should exist!
+                    _ => return Ok(()),        // should exist!
```

Comment:
Added a comment here. When you specify other sub-properties independently and try to serialize the shorthand, it crashes here instead of returning empty string.